### ISSUE TITLE
acceptInsecureCerts docs unclear about usage

### ISSUE THREAD

**Issue by drachels** [mdn/content#28588]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/acceptInsecureCerts

### What specific section or headline is this issue about?

acceptInsecureCerts

### What information was incorrect, unhelpful, or incomplete?

The info, to me, is incomplete! I am a newbie at this, and NOWHERE does it even hint, where and how to enter that python code snippet. I have looked at about a dozen similar pages, and everyone of them assumes that anyone reading it is going to know where to put it, but I do not.

### What did you expect to see?

A more complete solution that actually tells me where and how to make the needed changes.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/webdriver/capabilities/acceptinsecurecerts`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/acceptInsecureCerts
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/webdriver/capabilities/acceptinsecurecerts/index.md
* Last commit: https://github.com/mdn/content/commit/4a6dacf8c68925a8538585be3b2728bcb271241e
* Document last modified: 2023-07-24T23:58:21.000Z

</details>

**Comment by Josh-Cena** [comment]
You put it anywhere and you use `python` to execute it. You don't put it in any specific place. What are you trying to find?

**Comment by drachels** [comment]
I am trying to run Moodle behat tests on a server that uses a self signed certificate. As I mentioned, all the pages of "help" for this that I have looked at, have all assumed the viewer would know what to do with their code snippets. I finally found one that leads me to believe I can put it in my behat feature file. I just tried that and even though I did not see anything different when a web page popped up, I did get some usable information in my terminal window. I really appreciate the quick response.

**Comment by drachels** [comment]
Yes. While running tests, the Firefox browser pops up, shows the normal warnings about risk due to self signed certificate, and then goes away. On my latest attempt, where I tried putting your snippet in the feature files, I finally got some feedback that might be helpful.

**Comment by Josh-Cena** [comment]
Closing because the documentation is self-contained. It appears that you are trying to integrate the API with some 3rd party framework, which we unfortunately cannot teach.
